posts/templates/posts/post.html

```diff
@@ -0,0 +1,20 @@
+{% extends 'base_layout.html' %}
+
```

Comment:
All nested tags and block/for/if/with blocks need to be indented. Indentation must be consistent.

```
{% extends 'base_layout.html' %}
{% block content %}
  <h1>This is the post by You</h1>
  {% if messages %}
    <ul class="messages">
      {% for message in messages %}
        <li>{% if message.tags %} class="{{ message.tags }}"{% endif %} {{ message }}</li>
      {% endfor %}
    </ul>
  {% endif %}
  <div class="post">
    <a href="{% url 'posts:post' id=post.id  %}"><h2>{{ post.title }}</h2></a>
    <p> {{ post.body }} </p>
    <p><strong> {{ post.date }} </strong></p>
    <p><strong> {{ post.id }} </strong></p>
  </div>
{% endblock %}
```